LayoutTests/imported/w3c/web-platform-tests/css/cssom/serialize-values.html

```diff
@@ -140,11 +140,33 @@
       return null;
     }
 
-    function create_result(actual, expected) {
+    // A more elegant way of doing this would be better, but for now this exception list handles cases where "center" must be omitted.
+    var minimal_results = {
+        "background-position: 5% center": "5%",
+        "background-position: 0.5% center": "0.5%",
+        "background-position: -5% center": "-5%",
+        "background-position: -0.5% center": "-0.5%",
+        "background-position: 0px center": "0px",
+        "background-position: 1px center": "1px",
+        "background-position: 0.1em center": "0.1em",
+        "background-position: 0px center": "0px",
+        "background-position: -1px center": "-1px",
+        "background-position: -0.1em center": "-0.1em",
+        "background-position: left center": "left",
+        "background-position: center top": "top",
+        "background-position: center center": "center",
+        "background-position: center bottom": "bottom",
+        "background-position: right center": "right",
+    };
+
+    function create_result(propertyName, actual, expected) {
+      var key = propertyName + ": " + expected
+      if (key in minimal_results)
+        expected = minimal_results[key]
```

Comment:
Is this the same as:

```
if (propertyName == "background-position")
  expected = expected.replace("center", "").trim();
```

Reply:
No.

When both are identifiers we can omit center if it's first or second.

When one is an identifier and the other is a number we can omit center if it's second but not if it's first.

When one is a pair then we can't omit center even if it's after the pair.

Reply:
Ah right, makes sense.